extension-androidauto/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 # main
 ## Features ✨ and improvements 🏁
+* Make the SDK compatible with any version of Mapbox Maps v10. ([#1706](https://github.com/mapbox/mapbox-maps-android/pull/1706))
```

Comment:
This PR actually made the extension only compatible with maps sdk v10.5.0+, isn't it?

Reply:
Oh right i didn't update this yet thanks!